### DISCUSSION TITLE
enqueue_trial not working as expected

### DISCUSSION THREAD

**Discussion by TheLegendAli** [optuna/optuna#4894] [Q&A] [answered]
I want optuna to run some trials by some suggestions that my other functions give, however it looks like it doesn't not accept the parameters. here is a dummy code

` 
storage = optuna.storages.JournalStorage(optuna.storages.JournalFileStorage("./optuna_journal.log"))
study_name='test'
new_study = optuna.create_study(sampler=optuna.samplers.NSGAIISampler(),
                                    pruner=optuna.pruners.MedianPruner(n_warmup_steps=10, n_startup_trials=50),
                                    direction='maximize',
                                    study_name=study_name,
                                    #storage=storage_name)
                                    storage=storage)

func = lambda trial: print(trial.params)

new_study.enqueue_trial({'x': 0.1, 'y': 0.2})

new_study.optimize(func, n_trials=10, 
                           callbacks=[partial(self.early_stopping_check, early_stopping_rounds=500)],
                           show_progress_bar = True, n_jobs=-1)
`

the results are:
{}
{}
{}
{}
{}
{}
{}
{}
{}
{}

how can I make my function print the values of x and y that I pass to it

**Comment by nzw0301** [comment, marked as answer]
You need to suggest x and y in the objective.

**Comment by TheLegendAli** [comment]
i resolved it thanks
